packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.stories.ts

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/subscriptionFirstInvoice',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
+  'subscriptionFirstInvoice',
+  'Sent to inform a user that their payment is currently being processed.',
```

Comment:
Probably worth mentioning here too that this is only for their first invoice. Looks like after that they're sent the `subsequent` template.